### DISCUSSION TITLE
Is it possible to make this menus searchable, via a desktop applet? That would be awesome

### DISCUSSION THREAD

**Discussion by titojff** [linuxmint/discussions#119] [Ideas]
I don't know if it's possible, but would save tons of time.

![Screenshot from 2023-08-13 08-07-49](https://github.com/linuxmint/discussions/assets/5201100/f7933711-6af2-4654-a5cb-26578f49a4ec)

**Comment by fredcw** [comment]
Good idea but very difficult if not impossible to implement. Different apps have different ways of storing their menu structure, many in the compiled program code itself and thus virtually impossible to decode.
